src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralOptions.cs

```diff
@@ -2,19 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Immutable;
-using System.Composition;
-using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Options.Providers;
 
-namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral
+namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral;
+
+internal sealed class SplitStringLiteralOptions
 {
-    internal sealed class SplitStringLiteralOptions
-    {
-        public static PerLanguageOption2<bool> Enabled =
-            new(nameof(SplitStringLiteralOptions), nameof(Enabled), defaultValue: true,
-                storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.SplitStringLiterals"));
-    }
+    public static Option2<bool> Enabled = new("SplitStringLiteralOptions_Enabled", defaultValue: true);
```

Comment:
Quite unrelated to the changes in this PR.

Is this option missing in `AutomationObject`? I suppose that export/import functionality doesn't currently persist it correctly

Reply:
Possibly. Feel free to file an issue.